solr/contrib/ltr/src/java/org/apache/solr/ltr/ranking/LTRCollector.java

```
@@ -111,10 +112,8 @@ public TopDocs topDocs(int start, int howMany) {
         topRerankDocs = reRankRescorer.rescore(searcher,
             mainDocs, howMany);
       } catch (final IOException e) {
```

Just remove the try catch, no need to rethrow the same exception